spec/models/druid_version_zip_spec.rb

```diff
@@ -63,7 +106,7 @@
     let(:zip_path) { '/tmp/bj/102/hs/9687/bj102hs9687.v0001.zip' }
 
     it 'returns zip string to execute for this druid/version' do
-      expect(dvz.zip_command).to eq "zip -vr0X -s 10g #{zip_path} #{moab_version_path}"
+      expect(dvz.zip_command).to eq "zip -vr0X -s 10g #{zip_path} bj102hs9687/v0001"
```

Comment:
don't need `let(:moab_version_path)` anymore

Reply:
good point!

Reply:
Removed it.